docs/DeveloperGuide.md

```diff
@@ -6,33 +6,360 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### User Interface (UI) - Class Implementation
+![img.png](img.png) <br>
```

Comment:
This image may have incorrect syntax.
Is the Class symbol supposed to be shown in such a way?
The function named should also be pre-fixed with + or - to showcase whether they are public/private.

Reply:
Agree with the above comment, seems like the class diagram could be using non-standard UML symbols/syntax.